style.css

```diff
@@ -0,0 +1,130 @@
+html{
+    box-sizing: border-box;
+    height: 100%;
+}
+
+*,
+*::before,
+*::after{
+    box-sizing: inherit;
+    margin: 0;
+    padding: 0;
+}
+
+body{
+    align-items: center;
+    background: linear-gradient(320deg, #f670cf, #32defd); 
+    display: flex;
```

Comment:
body should have only general styles, that will not change. If you need display: flex, please make for it one more container